geoprocessing/src/modules/protected-areas/worker/__mocks__/shapefile-service.ts

```diff
@@ -0,0 +1,10 @@
+import { Injectable } from '@nestjs/common';
```

Comment:
worth adding this mock under the directory of the existing `ShapefilesModule` where we have the actual `ShapefilesService`?

Reply:
it it would be widely used, I would rather think to put it under `utils/__mocks__`. 
However, every module could need different behavior/mock. Not sure if we want to put all mocks around into bucket getting larger and larger 🤔

Reply:
moved to utils in https://github.com/Vizzuality/marxan-cloud/pull/203/commits/a4d224abc76db2fac79f38c0279ae848dd4eed67